From 7ae7c838b156ba1b6bf89ab60ba9c8df04532dd0 Mon Sep 17 00:00:00 2001
From: Guillaume Bougard <gbougard@teclib.com>
Date: Sat, 13 Apr 2019 18:47:36 +0200
Subject: [PATCH] fix: Add few minor fixes

* build: fix fusioninventory-remoteinventory was not installed
* inventory: fix error message while starting fusioninventory-inventory script
---
 Changes                                     | 8 ++++++++
 Makefile.PL                                 | 1 +
 lib/FusionInventory/Agent/Task/Inventory.pm | 2 +-
 3 files changed, 10 insertions(+), 1 deletion(-)

diff --git a/Changes b/Changes
index 83be862d0d..36849904a7 100644
--- a/Changes
+++ b/Changes
@@ -1,5 +1,13 @@
 Revision history for FusionInventory agent
 
+2.5.1 not relased yet
+
+core:
+* build: fix fusioninventory-remoteinventory was not installed
+
+inventory:
+* fix error message while starting fusioninventory-inventory script
+
 2.5 Fri, 12 Apr 2019
 
 core:
diff --git a/Makefile.PL b/Makefile.PL
index 38e6eebdca..71aae700c9 100644
--- a/Makefile.PL
+++ b/Makefile.PL
@@ -104,6 +104,7 @@ install_script 'bin/fusioninventory-wakeonlan';
 install_script 'bin/fusioninventory-netdiscovery';
 install_script 'bin/fusioninventory-netinventory';
 install_script 'bin/fusioninventory-esx';
+install_script 'bin/fusioninventory-remoteinventory';
 
 resources
     homepage    => "http://www.fusioninventory.org",
diff --git a/lib/FusionInventory/Agent/Task/Inventory.pm b/lib/FusionInventory/Agent/Task/Inventory.pm
index 84a3cafbda..8fb2cd8806 100644
--- a/lib/FusionInventory/Agent/Task/Inventory.pm
+++ b/lib/FusionInventory/Agent/Task/Inventory.pm
@@ -59,7 +59,7 @@ sub run {
         tag      => $tag
     );
 
-    $self->{logger}->info("New inventory from $self->{deviceid} for $self->{target}->{id}".
+    $self->{logger}->info("New inventory from ".$inventory->getDeviceId()." for $self->{target}->{id}".
         ( (defined($tag) && length($tag)) ? " (tag=$tag)" : "" ));
 
     # Set inventory as remote if running remote inventory like from wmi task
